src/java/picard/analysis/directed/InsertSizeMetricsCollector.java

```diff
@@ -121,7 +133,13 @@ public void addMetricsToFile(final MetricsFile<InsertSizeMetrics,Integer> file)
                 final double total = Histogram.getCount();
 
                 // Only include a category if it has a sufficient percentage of the data in it
-                if( total > totalInserts * minimumPct ) {
+                if( total >= totalInserts * minimumPct ) {
+                    // in the case that minimumPct == 0 and total == 0, just make an empty histogram with zero values for the entire range
+                    if (Histogram.isEmpty()) {
```

Comment:
Can you comment in the code on why this is necessary?  I assume it's to make the code below for the `WIDTH_OF...` not blow up, but I'm not sure.